README.md

```diff
@@ -25,27 +25,26 @@
 General information about the Web of Things can be found on https://www.w3.org/WoT/.
   
 ---
+The Web of Things (WoT) is made of entities (Things) that can describe their capabilities in a machine-interpretable Thing Description (TD) and expose these capabilities through the WoT Interface, that is, network interactions modeled as Properties (for reading and writing values), Actions (to execute remote procedures with or without return values) and Events (for signaling notifications).
 
-The main deliverable being developed in this repository is the [WoT Scripting API Specification](http://w3c.github.io/wot-scripting-api/).
+Scripting API is an optional building block in WoT and it is typically used in gateways or browsers that can run a WoT Runtime and script management, providing a convenient way to use WoT concepts in high-level applications.
 
-Releases for [published versions](https://www.w3.org/TR/wot-scripting-api/) are found in [releases](./releases/).
+This specification describes an application programming interface (API) representing the WoT Interface that allows scripts to discover, operate Things, and expose locally defined Things characterized by WoT Interactions specified by a script.
```

Comment:
I feel like “script” is a bit doubled here, maybe we can also omit the second occurrence?

On the other hand, although it introduces some redundancy, I had the feeling that something is missing after “discover”:

```suggestion
This specification describes an application programming interface (API) representing the WoT Interface that allows scripts to discover Things, operate Things, and expose locally defined Things characterized by WoT Interactions.
```

Reply:
Suggestions looks good 👍